traitcuration/traits/templates/traits/datasources.html

```diff
@@ -0,0 +1,16 @@
+{% extends "traits/base.html" %} 
+{% block title %} External Datasources {%endblock %} 
+{% block content %}
+<div class="uk-container uk-container-xlarge">
+  <a href="{% url 'fetch_data' %}"
+    ><button class="uk-button uk-button-primary">FETCH DATA</button>
+  </a>
+  {% if messages %}
+    <ul class="messages">
+        {% for message in messages %}
+        <li {% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message }}</li>
+        {% endfor %}
+    </ul>
+    {% endif %}
```

Comment:
Maybe I'm confused, but what is the `<ul class="messages">` block supposed to do? I don't see message.tags populated anywhere, and the `{% if messages %}` conditional never seems to fire.

Reply:
Yes, that's my bad, i was testing Django messages and forgot to remove that from the page